src/core/Microsoft.PowerToys.Settings.UI.UnitTests/ViewModelTests/General.cs

```diff
@@ -21,11 +23,56 @@ public class General
 
         private Mock<ISettingsUtils> mockGeneralSettingsUtils;
 
+
+
         [TestInitialize]
         public void SetUp_StubSettingUtils()
         {
             mockGeneralSettingsUtils = ISettingsUtilsMocks.GetStubSettingsUtils<GeneralSettings>();
         }
+		
+        /// </summary>
+        [TestMethod]
+        [DataRow("v0.18.2", "settings.json")]
+        [DataRow("v0.19.2", "settings.json")]
+        [DataRow("v0.20.1", "settings.json")]
+        [DataRow("v0.21.1", "settings.json")]
+        [DataRow("v0.22.0", "settings.json")]
+        public void OriginalFilesModificationTest(string version, string fileName)
+        {
+            var mockGeneralIOProvider = BackCompatTestProperties.GetGeneralSettingsIOProvider(version);
+            var mockGeneralSettingsUtils = new SettingsUtils(mockGeneralIOProvider.Object);
+            GeneralSettings originalGeneralSettings = mockGeneralSettingsUtils.GetSettings<GeneralSettings>();
+            var generalSettingsRepository = new BackCompatTestProperties.MockSettingsRepository<GeneralSettings>(mockGeneralSettingsUtils);
+
+            // Initialise View Model with test Config files
+            // Arrange
+            Func<string, int> SendMockIPCConfigMSG = msg => { return 0; };
+            Func<string, int> SendRestartAdminIPCMessage = msg => { return 0; };
+            Func<string, int> SendCheckForUpdatesIPCMessage = msg => { return 0; };
+            GeneralViewModel viewModel = new GeneralViewModel(
+                generalSettingsRepository,
+                "GeneralSettings_RunningAsAdminText",
+                "GeneralSettings_RunningAsUserText",
+                false,
+                false,
+                UpdateUIThemeMethod,
+                SendMockIPCConfigMSG,
+                SendRestartAdminIPCMessage,
+                SendCheckForUpdatesIPCMessage,
+                string.Empty);
```

Comment:
[nit] Can we use named args for this particular declaration since there are many arguments?